docs/team/lfrostbytee.md

```diff
@@ -5,28 +5,43 @@ title: Ong Wee Marcus's Project Portfolio Page
 
 ### Project: Duke The Market
 
-Duke The Market is a desktop contact management system application 
-used for managing customer contacts in department stores. The user interacts with it using a CLI, 
-and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+Duke The Market is a desktop application used for managing customer contacts and events in department stores. 
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. 
+It is written in Java, and has about 10 kLoC.
 
 Given below are my contributions to the project.
 
-* **Code contributed**: to be added soon
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=lfrostbytee&breakdown=true)
 
 * **Enhancements implemented**:
-  * to be added soon
+    * Updated the GUI for the application (Pull requests [\#54](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/54), [\#131](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/131))
+    * Added the `listEvents` command [\#54](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/54)
+        * What it does: displays all events in the application to the user.
+        * Justification: This feature allows the user to see events in the current order in which they are stored as data and can be extended to sort events permanently.
+    * Added the `Date` and `Time` attribute for Events and Persons [\#106](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/106), [\#129](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/129)
+        * What it does: events can now store a starting date and time while each person can store their date of birth.
+        * Justification: This feature allows the user to see the age statistics for each event and see what time an event begins.
```

Comment:
Hmm was thinking that since this is a sub-point of introducing 'Date' and 'Time' class we shouldn't talk about statistics at this stage and leave it for later? Or perhaps can leave it at the end?

```suggestion
        * Justification: This feature allows the user to view the start date and time of every event in the application and also view the date of birth of all persons in the application. These attributes will eventually allow for event attendees statistics such as gender distribution to be displayed on a pie chart.
```